server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@koushik-das : Yes, We give priority to the project, if not specified then checking the account.